setup.py

```diff
@@ -2,7 +2,8 @@
 
 
 def listify(filename):
-    return [line for line in open(filename, 'r').read().split('\n') if line]
+    with open(filename, 'r') as f:
```

Comment:
Why this change?

Reply:
So that the file object doesn't remain open